views/news/article.hbs

```diff
@@ -1,12 +1,15 @@
 {{#extend "news/news"}}
 {{#content "page"}}
 <p>
-    <small class="text-muted">{{timeFromNow news.displayAt}}</small>
+    <small class="text-muted">{{$t "news.text.created" }}</small>
     {{#unless isRSS}}
     <small class="text-muted">
-        {{$t "news.text.from" }} {{news.creator.firstName}} {{news.creator.lastName}} {{$t "news.text.created" }}
+        {{#if news.creator}}
+        {{$t "news.text.by" }}{{news.creator.firstName}} {{news.creator.lastName}}
+        {{/if}}
+        {{timeFromNow news.displayAt}}
         {{#if news.updater}}
-        {{$t "news.text.lastFrom" }} {{news.updater.firstName}} {{news.updater.lastName}} {{$t "news.text.processed" }}
+        {{$t "news.text.last" }} {{$t "news.text.processed" }} {{$t "news.text.by" }}{{news.updater.firstName}} {{news.updater.lastName}}{{$t "news.text.closingBracket" }}
         {{/if}}
```

Comment:
can you find other solution instead of opening and closing bracket in translations? Why not in code directly?
no reason to have 3 consecutive translation labels which are not used anywhere else

Reply:
i fixed it as agreed with ux